doc/ReleaseNotes/_ReleaseNotes.md

```diff
@@ -72,6 +72,7 @@
 * [WASM] Improve element arrange and transform performance
 * Restore original SymbolIcon.SymbolProperty as a C# property
 * Add support for `MediaPlaybackList`
+* Update Uno.SourceGenerationTasks to improve build performance
```

Comment:
I think we can add more here :) 
Your PR description of the new behavior was very good, maybe we could add it here as well ?
```
Move to the latest Uno.SourceGenerationTasks to improve project parsing performance, and allows for the removal of unused targets caused by nventive/Uno.SourceGeneration#2. Uno.Xaml and Uno.UI.BindingHelpers now only build the required targets.
Move to net461 for test projects so the the Uno.Xaml project can be referenced properly
Use the latest MSBuild.Sdk.Extras for actual parallel cross-targeted builds
Move the nuget package versions to the Directory.Build.targets file so it's easier to change all versions at once.
```